impeller/renderer/backend/gles/sampler_gles.cc

```diff
@@ -53,7 +53,8 @@ static GLint ToParam(MinMagFilter minmag_filter,
   FML_UNREACHABLE();
 }
 
-static GLint ToAddressMode(SamplerAddressMode mode) {
+static GLint ToAddressMode(SamplerAddressMode mode,
```

Comment:
nit: I might just pass in a bool for whether or not decal is supported.

Reply:
Just a drive by comment; Is it just me or does anyone else find "decal" mode to be a non-standard and/or overly confusing term? How about just `ClampToBorderColor`? Thats what the various APIs call it (`GL_CLAMP_TO_BORDER_EXT`, `MTLSamplerAddressModeClampToBorderColor`, `VK_SAMPLER_ADDRESS_MODE_CLAMP_TO_BORDER`).

Perhaps later, we can rename this enum value for consistency.

Reply:
There is a cost for non-consistency with the underlying graphics API, but there is also a value in matching the framework level enumeration which uses decal as a terminology. I don't feel as strongly about decal as I do about "FilterQuality", which is entirely opaque - at least decal has some semantic meaning

Reply:
> nit: I might just pass in a bool for whether or not decal is supported.

Done